utils/downloadFile.ts

```diff
@@ -0,0 +1,49 @@
+import fs from 'node:fs';
+import { join, parse } from 'node:path';
+import sharp from 'sharp';
+
+const scandicRegex = /[ÄäÖöÅå]/g;
+const scandicCharMap:Record<string, string> = {
+  Ä: 'A',
+  ä: 'a',
+  Ö: 'O',
+  ö: 'o',
+  Å: 'a',
+  å: 'a'
+};
+
+export const replaceScandicCharacters = (text: string) =>
+  text.replace(scandicRegex, char => scandicCharMap[char]);
```

Comment:
Mites ois vähäsen robustimpi vaihtoehto tälle:

```typescript
const s = "John Smith's Cool Page";
const filename = s.replace(/[^a-z0-9]/gi, '_').toLowerCase();
```

https://stackoverflow.com/questions/8485027/javascript-url-safe-filename-safe-string

Reply:
Hah en mä ees käyttänyt sitä 😅 Noi ääkköset ei lokaalisti ainakaan tuota ongelmia niin ehkä nyt jätän ne sellaseks. Mä yhdistäisin sun ja mun ratkaisun (eli kaks regexia peräkkäin), koska sit kuvien nimet pysyis vähän luettavimpana. Pelkästään tolla alemmalla menis "konstan jytä äijä radio" -> "konstan_jyt__ij_radio" 

Mut jos ei tuu ongelmia niin skippaan ton korvauksen kokonaan